azurerm/azurerm_kubernetes_cluster/simple/main.tf

```diff
@@ -3,7 +3,7 @@
 
 # Documentation: https://www.terraform.io/docs/language/settings/index.html
 terraform {
-  required_version = ">= 0.14.0"
+  required_version = ">= 1.0.0"
```

Comment:
terraform apply failed here in CI:
* https://github.com/ContainerSolutions/terraform-examples/runs/3281538792?check_suite_focus=true#step:6:5309

It seems to be some leaked resources – resource group existed in Azure, probably not deleted in some previous runs.


```
Plan: 2 to add, 0 to change, 0 to destroy.
azurerm_resource_group.changeme_simple_k8s_rg: Creating...
╷
│ Error: A resource with the ID "/subscriptions/***/resourceGroups/changeme-simple-k8s-rg" already exists - to be managed via Terraform this resource needs to be imported into the State. Please see the resource documentation for "azurerm_resource_group" for more information.
│ 
│   with azurerm_resource_group.changeme_simple_k8s_rg,
│   on main.tf line 30, in resource "azurerm_resource_group" "changeme_simple_k8s_rg":
│   30: resource "azurerm_resource_group" "changeme_simple_k8s_rg" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/azurerm/azurerm_kubernetes_cluster/simple
There are some problems with the CLI configuration:
╷
│ Error: Error parsing /home/runner/.terraformrc: At 3:16: illegal char
│
╵

As a result of the above problems, Terraform may not behave as intended.
```

Reply:
I've just deleted the leaked resource group manually in Azure.